content/administration/upgrade.rst

```diff
@@ -156,6 +156,17 @@ project <https://odoo.sh/project>`_.
 
          $ python <(curl -s https://upgrade.odoo.com/upgrade) test -d <your db name> -t <target version>
 
+      .. note::
+         This command has some requirements on the environment it runs in:
+
+         - Some external commands. It will complain if one is missing.
```

Comment:
```suggestion
         - Some external commands. An error will be displayed if one is missing.
```
"Some external commands" is quite vague. Could it be more precise?

Reply:
I don't want to list them here, I tried to at least make more clear where they should come from.